sample/bin/constants.js

```diff
@@ -0,0 +1,33 @@
+const chalk = require('chalk');
+const path = require('path');
+
+const startScript = 'parcel src/index.html';
+const PACKAGE_NAME = 'bigtest-sample';
+const TARGET_DIR = `${process.cwd()}/${PACKAGE_NAME}`;
+const SOURCE_DIR = path.dirname(__dirname);
```

Comment:
Let's inline these as local variables. They aren't overridable, and so having them in a `constants` file only serves to remove them from the context in which they're used, making it hard for the reader to understand.

having the messages in a separate file can make sense though